src/main.cpp

```diff
@@ -99,6 +100,7 @@ int main ( int argc, char** argv )
         {
             bIsClient = false;
             tsConsole << "- server mode chosen" << endl;
+            CommandLineOptions += "--server";
```

Comment:
Should we only set the strings for command line arguments which are actually used in the settings class? I actually prefer to add them all as you did but the other solution would give less code and would make clear that the string is actually used. What is your opinion?

Reply:
I think there should be "greater equality" between command line and GUI - at the moment, there's a lot that the UI can do (particular the client) that can't be set on start up from the command line.  I'd also like to see most if not all of what can be set from the command line available in the UI.  And then the UI settings should all get loaded / saved to the inifile.

So it seemed logical to help that process by putting this in consistently, at least.

Reply:
Ok, agreed.